services/src/main/java/io/grpc/services/BinaryLogProviderImpl.java

```diff
@@ -52,14 +52,22 @@ public BinaryLogProviderImpl() {
   @Nullable
   @Override
   public ServerInterceptor getServerInterceptor(String fullMethodName) {
-    return factory.getLog(fullMethodName).getServerInterceptor(getServerCallId());
+    BinaryLog log = factory.getLog(fullMethodName);
+    if (log == null) {
+      return null;
+    }
+    return log.getServerInterceptor(getServerCallId());
   }
 
   @Nullable
   @Override
   public ClientInterceptor getClientInterceptor(
       String fullMethodName, CallOptions callOptions) {
-    return factory.getLog(fullMethodName).getClientInterceptor(getClientCallId(callOptions));
+    BinaryLog log = factory.getLog(fullMethodName);
```

Comment:
What happens if ```fullMethodName``` is null? Would a null check help here?

Reply:
It would be a good defensive check. I'll add one.